### DISCUSSION TITLE
ST7789 lcd

### DISCUSSION THREAD

**Discussion by frank-que** [probonopd/MiniDexed#289] [Ideas]
Hello, 

I recently purchased this item: [1.54inch Game LCD](https://spotpear.com/index/study/detail/id/65.html) (at $15.89)

CIRCLE supports ST7789 and I successfully tested and  compiled the driver demo. It was only necessary to specify the reset pin which is 27.

It would be interesting if it can be used as a graphics controller, with the joystick and a few of the keys instead of the rotary encoder and the LCD 1602 display.

In relation to the cost of the above item, it might be an interesting idea to consider.

Regards,
Frank

**Comment by diyelectromusic** [comment]
Graphical displays would be nice, but would also be a fairly significant piece of reworking of the UI everything currently assumes it is driving a "text" output supported by the circle environment.

Kevin
